code/modules/mob/living/living_defines.dm

```diff
@@ -33,6 +33,9 @@
 	///Damage caused by being cloned or ejected from the cloner early. slimes also deal cloneloss damage to victims
 	var/cloneloss = 0
 
+	/// Rate at which fire stacks should decay from this mob
+	var/fire_stack_decay_rate = -0.05
```

Comment:
This is the rate at which carbons decayed firestacks at, right? just making sure because i'm getting a slight case of the heebies looking at all of this excised code